main/src/vtr_navigation/launch/vtr.launch.py

```diff
@@ -19,7 +19,7 @@ def generate_launch_description():
         "namespace": 'vtr',
         "executable": 'vtr_navigation',
         "output": 'screen',
-        #"prefix": 'xterm -e gdb -ex run --args',
```

Comment:
Revert to previous.

Reply:
done

Reply:
By previous I meant that it should be commented out.